processor/samplingprocessor/tailsamplingprocessor/sampling/counting_sampler.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2019 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampling
+
+import (
+	"math/rand"
+	"sync/atomic"
+	"time"
+)
+
+//DIVISOR : Divisor constant
+const DIVISOR = 100
```

Comment:
is precision sufficient? I would think that for high QPS environment if may need precision of 0.001 if not higher.

Reply:
Hmm. Maybe we can make this configurable. I will ensure to externalize this in my next PR.